### PR TITLE
started main Crop and Soil wiki page

### PR DESCRIPTION
**Pull request by morrowcj**
# Summary
This PR creates the main wiki for the Crop and Soil module. The high-level summary (what and why) are included but the remaining sections will be completed at a later time (as the module and documentation progresses.

# Note
There are a few `TODO` comments embedded in the raw text of the markdown files. These `TODO`s will remain to remind us to update this document.

### CODE REVIEW

File: SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Crop and Soil
+
+The **Crop and Soil** submodule of the **RuFaS** whole-farm system model simulates agricultural fields, the soils they
+contain, the crops grown in them, and utilization by animals 
+(i.e., grazing). <!-- TODO: grazing not yet implemented --> The simulations represent real biogeophysical 
+processes that occur on farms and can assist growers in making decisions about their systems and management 
+practices.
+
+Users define farms that contain a number of fields. Each field contains soil whose structure and composition will
+change throughout the course of the simulation: nutrients can be added to the soil through amendments and tillage;
```
Comment (LaplacetteCM) [inline SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md:11]: Just an idea: what would you think about sharing here a user-friendly diagram/image to help with the "what"? farm --> fields --> options in the field --> temporal and spatial scales --> dynamic processes represented and outputs of interest. 

I remember your presentation in the annual meeting when explaining the redesign where you puntualized, in other words, that "field" was like the unit of study/spatial scale where all the processes were assesed. I would emphatise it.

File: SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The **Crop and Soil** submodule of the **RuFaS** whole-farm system model simulates agricultural fields, the soils they
+contain, the crops grown in them, and utilization by animals 
+(i.e., grazing). <!-- TODO: grazing not yet implemented --> The simulations represent real biogeophysical 
+processes that occur on farms and can assist growers in making decisions about their systems and management 
+practices.
+
+Users define farms that contain a number of fields. Each field contains soil whose structure and composition will
+change throughout the course of the simulation: nutrients can be added to the soil through amendments and tillage; 
+water and nutrients cycle through the system, affected by weather; soils can erode and compact depending on their
+usage. A field can be planted with crops (one or multiple species), which photosynthesize and exchange resources with
```
Comment (LaplacetteCM) [inline SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md]: "A field can be planted with annual or perennial crops" ?

File: SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+the soil to grow. 
+
+The field systems and their driving processes are simulated on a daily basis. This fine-scale temporal
+resolution allows for systems that can rapidly respond to external changes (i.e., weather and management). Users can 
+define management schedules that determine when crops are planted and harvest, when manure or fertilizers are added
+to the soil, when and what kind of tillage occurs, and the timing and duration of grazing. All these processes
+contribute to the resulting estimates of interest. Some such output that the model simulates includes biomass and 
+nutrient composition of crops throughout their life cycle, harvestable economic yield produced by the crops at the end
+of a growing season, soil health and composition, soil nutrient retention and loss, 
+environmental impacts, <!-- TODO: emmissions and other environmental factors not yet implemented --> and the
```
Comment (LaplacetteCM) [inline SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md:25]: I understand where you go with environmental impacts, but nothing was mentioned in the text above, so might sound a little wide if we don't clarify the way we are addressing it.

File: SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md (hunk 1 above)
Comment (morrowcj) [reply SC_redesign/Crop_and_Soil/docs/Crop_and_Soil.md:11]: good idea!

**Review by LaplacetteCM — COMMENTED** [review]
Great idea to provide a summary of the module and the way it works. I think it clearly explains the what and the why but also will be important as a basis for accessing the files of interest.

**Pull request merged**